Libraries/Core/InitializeCore.js

```diff
@@ -173,6 +173,8 @@ defineProperty(global, 'Headers', () => require('fetch').Headers);
 defineProperty(global, 'Request', () => require('fetch').Request);
 defineProperty(global, 'Response', () => require('fetch').Response);
 defineProperty(global, 'WebSocket', () => require('WebSocket'));
+defineProperty(global, 'Blob', () => require('Blob'));
+defineProperty(global, 'URL', () => require('URL'));
```

Comment:
How important is that really? We're fighting really hard to reduce bridge startup time, and all of these global modules reduce the amount of work we can do on-demand / lazily.